### ISSUE TITLE
Put all services inside docker-compose

### ISSUE THREAD

**Issue by Tyrrrz** [dotnet/nuget-trends#147]
Currently, only postgres and rabbitmq are included in `docker-compose.yml`. It would be **extremely** convenient for new contributors if that also contained frontend/backend/worker services as well.

**Comment by joaopgrassi** [comment]
I guess it depends on which part of the project you are working on. If you are working on the front-end, I only need the db+api. So maybe we could have multiple compose files that makes sense for each contributor "area"?

**Comment by Tyrrrz** [comment]
You can actually have one compose file, but then do `docker-compose up service1 service2` to selectively run services, it's quite convenient.

**Comment by joaopgrassi** [comment]
Yep, that also works. One "downsize" of this approach is that I have to remember to type all the services I need so :smile:. Wheres if we have the individual `compose-spa` then I can just run that. Lazy I know..

**Comment by Tyrrrz** [comment]
You could probably also implement `compose-without-frontend` and `compose-without-backend` as batch files/shell scripts.

**Comment by joaopgrassi** [comment]
Good idea! That would be the best of both worlds then :).

**Comment by bruno-garcia** [comment]
we're using Aspire now
